.github/workflows/release-please.yml

```
@@ -0,0 +1,24 @@
+on:
+    push:
+      branches:
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)